production/ksonnet/loki/config.libsonnet

```diff
@@ -277,9 +329,9 @@
       // Default schema config is bigtable/gcs, this will need to be overridden for other stores
       schema_config: {
         configs: [{
-          from: '2018-04-15',
-          store: 'bigtable',
-          object_store: 'gcs',
+          from: '%s' % $._config.schema_start_date,
```

Comment:
if there is no default for this variable ( fair enough ) should it be an `error 'must define schema start_date'` ?

Reply:
on line 172 we have this variable: `schema_start_date: '2018-07-11',` because this i keep it as is.